api/client_test.go

```diff
@@ -527,7 +527,7 @@ func (s *clientSuite) TestSetModelAgentVersionDuringUpgrade(c *gc.C) {
 	_, err = s.State.EnsureUpgradeInfo(machine.Id(), agentVersion, nextVersion)
 	c.Assert(err, jc.ErrorIsNil)
 
-	err = s.APIState.Client().SetModelAgentVersion(nextVersion)
+	err = s.APIState.Client().SetModelAgentVersion(nextVersion, false)
```

Comment:
I appreciate that this is an old, full-stack test. It'd be great to make it leaner \o/ Especially since the only thing we'd want to check here is that all the parameters are passed in.